src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -107,7 +107,7 @@ namespace AppInstaller::CLI::Workflow
         }
         catch (...)
         {
-            context.Reporter.Error() << "Failed to open the source; try removing and re-adding it" << std::endl;
+            context.Reporter.Error() << Resource::String::OpenSourceFailed << std::endl;

```

Comment:
>OpenSourceFailed [](start = 58, length = 16)

Oh no, we both did this :)

Reply:
:)

---
In reply to: [499904348](https://github.com/microsoft/winget-cli/pull/588#discussion_r499904348) [](ancestors = 499904348)